source/adapters/level_zero/v2/queue_immediate_in_order.cpp

```diff
@@ -11,8 +11,51 @@
 #include "queue_immediate_in_order.hpp"
 
 namespace v2 {
+
+static int32_t getZeOrdinal(ur_device_handle_t Device, queue_group_type type) {
+  if (type == queue_group_type::MainCopy && Device->hasMainCopyEngine()) {
+    return Device->QueueGroup[queue_group_type::MainCopy].ZeOrdinal;
+  }
+  return Device->QueueGroup[queue_group_type::Compute].ZeOrdinal;
+}
+
+static std::optional<int32_t> getZeIndex(const ur_queue_properties_t *Props) {
+  if (Props) {
+    if (Props->pNext) {
+      const ur_base_properties_t *extendedDesc =
```

Comment:
I noticed that the v2 code doesn't follow a consistent variable naming convention, e.g. we start with a lowercase letter here but a few lines on there's a variable defined starting with an upper case letter.

There's quite a few cases of this so maybe not something for this PR, but could be good to decide on a consistent format while v2 is still relatively new.

Reply:
Yeah, good point. I copied part of this from different places from L0 adapter but I agree it would be better to have a consistent style. I updated all variable names (that I introduce in this PR) to start with lowercase letter. I will also do a follow-up PR to make naming in command_list_cache.*pp consistent.